### PR TITLE
Add PyTorch demo

### PR DESCRIPTION
**Pull request by ClawSeven**

### CODE REVIEW

File: demos/pytorch/demo.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    # all learnable parameters in the model.
+    loss.backward()
+
+    # Update the weights using gradient descent. Each parameter is a Tensor, so
+    # we can access its gradients like we did before.
+    with torch.no_grad():
+        for param in model.parameters():
+            param -= learning_rate * param.grad
+
+print("Done")
```
Comment (guzongmin) [inline demos/pytorch/demo.py:56]: Solve the "No newline at end of file" issue.

Comment (ClawSeven) [reply demos/pytorch/demo.py]: Done~

File: demos/README.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ This set of demos shows how real-world apps can be easily run inside SGX enclave
 * [sqlite](sqlite/) A demo of [SQLite](https://www.sqlite.org) SQL database engine.
 * [tensorflow_lite](tensorflow_lite/): A demo and benchmark of [Tensorflow Lite](https://www.tensorflow.org/lite) inference engine.
 * [xgboost](xgboost/): A demo of [XGBoost](https://xgboost.readthedocs.io/en/latest).
+* [pytorch](pytorch/): A demo of [Pytorch](https://pytorch.org/).
```
Comment (liqinggd) [inline demos/README.md:27]: Please put the pytorch demo before redis, the demos are sorted alphabetically.

Comment (ClawSeven) [reply demos/README.md]: Done

**Comment by liqinggd** [comment]
Partially implements https://github.com/occlum/occlum/issues/386

**Pull request merged**